### ISSUE TITLE
setup script is too lsstsw specific

### ISSUE THREAD

**Issue by danielsf** [DarkEnergyScienceCollaboration/Twinkles#188]
The setup.sh script only works for users who are using lsstsw (as opposed to eups distrib install or conda) to install their copy of the stack.  Specifically, I am referring to this line
```
export EUPS_PATH=$inst_dir:${LSSTSW}/stack
```
EUPS_PATH will be declared by loadLSST.bash for users who are using eups distrib install.

$LSSTSW/stack will not exist for users who are not using lsstsw.

I wonder if the setup script should just declare Twinkles to EUPS.  Users would then be responsible/able to make sure that EUPS is setup (however that is done on their particular stack install) before running the Twinkles setup script.

**Comment by jchiang87** [comment]
Yep, I agree.    The setup.sh and setup.csh scripts were from before we sorted out my understanding of eups usage during the DESC meeting.   I'm thinking now that we should just have instructions on how to declare Twinkles to eups and forgo any setup script that does it in a particular way.

**Comment by danielsf** [comment]
I agree that documentation is probably the way to go on this.
